scripts/ci/upload-build

```diff
@@ -2,35 +2,34 @@
 set -ex
 
 # configuration
+AWS_BUCKET='downloads.mesosphere.io'
+FILENAME='dcos-ui'
 BUILD_VARIANT=${BUILD_VARIANT:-''}
 BRANCH_NAME=${BRANCH_NAME:-'master'}
-DIST_PATH=${DIST_PATH:-'dist'}
-PACKAGE_PATH=${PACKAGE_PATH:-'.'}
-SHA_SLUG=${SHA_SLUG:-$(git rev-parse --short HEAD)}
+PKG_VERSION=$(git tag -l --contains HEAD)
 
-# aws settings
-AWS_BUCKET='downloads.mesosphere.io'
-BUCKET_PATH='dcos-ui'
+if [[ $PKG_VERSION ]]
+then
+  PKG_VERSION="-$PKG_VERSION"
+fi
 
-# build release filename
-FILENAME='dcos-ui'
-PKG_VERSION=$(python -c "import sys, json; \
-  pkg = json.load(open('$PACKAGE_PATH/package.json')); \
-  sys.stdout.write(str(pkg['version']));")
 PKG_BRANCH=$(echo ${BRANCH_NAME} | sed -E 's/([^/]*)$|./\1/g')
-RELEASE_NAME="${PKG_BRANCH}+${FILENAME}${BUILD_VARIANT}-v${PKG_VERSION}+${SHA_SLUG}.tar.gz"
+
+# ex. master+dcos-ui-v1_12_0-rc_1.tar.gz
+# ex. master+dcos-ui-ee-v1_12_0-rc_1.tar.gz
+RELEASE_NAME="${PKG_BRANCH}+dcos-ui${BUILD_VARIANT}${PKG_VERSION}.tar.gz"
 
 # tar dist
-cd "$DIST_PATH"
+cd "dist"
 tar czf "../$RELEASE_NAME" .
 cd ..
+
 BUILD_SHA=$(shasum ${RELEASE_NAME} | cut -d " " -f1)
 ENCODED_RELEASE_NAME=$(echo ${RELEASE_NAME} | sed "s/\+/\%2B/g")
-DOWNLOAD_URL=$(echo "https://${AWS_BUCKET}/${BUCKET_PATH}/${ENCODED_RELEASE_NAME}")
-FILENAME='buildinfo.json'
+DOWNLOAD_URL=$(echo "https://${AWS_BUCKET}/dcos-ui/${ENCODED_RELEASE_NAME}")
```

Comment:
(Optional) Why did you inline the `$BUCKET_PATH` 🤔

Reply:
Fixed.